Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -70,6 +70,9 @@ def bottle_args
              depends_on:  "--write",
              description: "When passed with `--write`, a new commit will not generated after writing changes "\
                           "to the formula file."
+      switch "--no-tab",
+             depends_on:  "--json",
+             description: "When passed with `--json`, the tab will be written to the JSON file but not the bottle."
```

Comment:
I find the name of this a bit confusing. It's named "no tab" but there is actually a tab written.

Reply:
@Bo98 Not into the bottle, though. Suggested rename?

Reply:
I'm not good with names but it should mention the bottle, like `--no-bottle-tab` or something.

Reply:
@Bo98 Given the command is called `bottle` the bottle part seems a bit redundant?

Reply:
`--external-tab`? A bit less convenient than `--no-tab` though.